server_ui/templates/confirm.html

```diff
@@ -12,20 +12,20 @@
   $('#waiting_div').show();
 }
 </script>
-<h1>Confirm your Vote</h1>
+<h1>Potvrdit vaše hlasování</h1>
 
```

Comment:
confirm je rozkazovaci zpusob. 
vase se bud pise s velkym nebo mas tykat (nutno se podivat, jak je to jinde)
vote je s velkym pismenem schvalne a imho ho prekladame jako "hlas"